src/k8s/pkg/component/metrics_server.go

```diff
@@ -0,0 +1,45 @@
+package component
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/canonical/k8s/pkg/snap"
+)
+
+func EnableMetricsServerComponent(ctx context.Context, s snap.Snap) error {
+	manager, err := NewHelmClient(s, nil)
+	if err != nil {
+		return fmt.Errorf("failed to get component manager: %w", err)
+	}
+
+	values := map[string]any{
```

Comment:
can we set a releaseName here so that this does not deploy as `ck-metrics-server`?